website/docs/r/registry_module.html.markdown

```diff
@@ -135,6 +168,8 @@ The `vcs_repo` block supports:
   and repository in your VCS provider. The format for Azure DevOps is `<ado organization>/<ado project>/_git/<ado repository>`.
 * `oauth_token_id` - (Optional) Token ID of the VCS Connection (OAuth Connection Token) to use. This conflicts with `github_app_installation_id` and can only be used if `github_app_installation_id` is not used.
 * `github_app_installation_id` - (Optional) The installation id of the Github App. This conflicts with `oauth_token_id` and can only be used if `oauth_token_id` is not used.
+* `branch` - (Optional) The git branch used for publishing when using branch-based publishing for the registry module.
+* `tags` - (Optional) Specifies whether tag based publishing is enabled for the registry module.
```

Comment:
add a statement to note that only one of `branch` or `tags` should be set but not both as per the API doc.

Reply:
Updated